src/components/Graph/GraphPreviewGrid.jsx

```diff
@@ -0,0 +1,40 @@
+import { Col, Row } from 'antd';
+import GraphPreview from './GraphPreview';
+import { TEXTS } from '../../constants/texts';
+import './GraphPreviewGrid.css';
+
+const GraphPreviewGrid = () => {
+  return (
+    <div className="centered-container">
+      <Row className="graph-row">
+        <Col className="graph-col" xs={24} xl={4}>
+          <GraphPreview text={TEXTS.DIRECTED_GRAPH} />
+        </Col>
+        <Col className="graph-col" xs={24} xl={4}>
+          <GraphPreview text={TEXTS.CIRCLE_GRAPH} />
+        </Col>
```

Comment:
Please don`t repeat the code, extract the common part and use it:

```js
const GraphCol = ({ text }) => (
  <Col className="graph-col" xs={24} xl={6}>
    <GraphPreview text={text} />
  </Col>
);

const GraphPreviewGrid = () => {
  return (
    <div className="centered-container">
      <Row className="graph-row">
        {graphData.map((item, index) => (
          <GraphCol key={index} text={item.text} />
        ))}
      </Row>
    </div>
  );
};

```

Also please use the `List` component that we already have in the app